xml/art_solution_automation.xml

```diff
@@ -0,0 +1,438 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="file:/usr/share/xml/geekodoc/rng/geekodoc5-flat.rnc"
+   type="application/relax-ng-compact-syntax"?>
+
+<!DOCTYPE article
+[
+  <!ENTITY % entities SYSTEM "generic-entities.ent">
+    %entities;
+]>
+<!--
+Confluence page:
+https://confluence.suse.com/x/8IEnGw
+-->
+
+<article version="5.1" xml:lang="en" xml:id="art-sol-automation"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+  <title>SAP Automation</title>
+  <info>
+    <abstract>
+      <para>
+        <remark>TBD</remark>
+      </para>
+    </abstract>
+  </info>
+
+  <note>
+    <title>No Official &suse; Support for &terraform;</title>
+    <para>
+      The referenced &salt; formulas takes care of the installation of
+      &sles4sapreg;, its configuration, as well as the configuration of
+      our high availability cluster.
+      The other parts belonging to the infrastructure like machines, network,
+      disk formats, mounting points etc. needs to be done with &suma;, &ay;,
+      or &terraform;. Keep in mind, our &terraform; project is published
+      <emphasis>as-is</emphasis> but not officially supported by &suse;.
+    </para>
+  </note>
+
+  <sect1 xml:id="sec-sol-autom-concept-overview">
+    <title>Conceptual Overview</title>
+    <para>
+      Starting from the idea to modernize and simplify &hana; deployments,
+      &suse; engineering worked on rewriting the deployment wizards in &salt;.
+      &salt; is a configuration management system that is also supported by
+      &susemgr; for any sort of automation.
+    </para>
+    <para>
+      The major motivation was to improve and simplify the installation of &hana;
+      on &sls;.
+    </para>
+    <remark>toms 2020-06-10: Mention Stack Formulas?</remark>
+  </sect1>
+
+  <sect1 xml:id="sec-sol-autom-terms">
+    <title>Terminology</title>
+    <remark>toms 2020-06-23: add more terms and further explanations?</remark>
+    <variablelist>
+      <varlistentry>
+        <term>Python</term>
+        <listitem>
+          <para>
+            A scripting language. It interacts with the low layers utilities
+            as &hana; and &crmshell; commands.
```

Comment:
It doesn't interact only with HANA. It interacts with multiple SAP tools (HANA management tools included)

Reply:
Ok, I've rewrote the sentence a bit:

    It interacts with the low layers utilities like crm shell and several SAP commands,
    including HANA management tools.

Reply:
Much better! We could list some of the commands, but we always have the risk that they might be changed in the future, so maybe better leave as it is

Reply:
Yeah, I thought of that too. However, exactly for the reason you've stated, I didn't include it. As I've used the little word "like", it gives just some examples. It means, it's just an example, there are more "low layer utilities" available. :wink: